setuptools_scm/git.py

```diff
@@ -4,7 +4,7 @@
 import warnings
 
 FILES_COMMAND = 'git ls-files'
-DEFAULT_DESCRIBE = 'git describe --tags --long --match *.*'
+DEFAULT_DESCRIBE = 'git describe --tags --long --dirty --match *.*'
```

Comment:
I wasn't understanding @RonnyPfannschmidt in the way that `--dirty` already used to be there, but did cause problems, and thus was removed. I thought that was just a general remark. But I'll investigate.